<!--AB_PersonalWebsite.html-->
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <title>Andrea Brigliadori</title>
    <link rel="stylesheet" href="Style/AB_PersonalWebsite.css">
<body>
    <div class="my-showcase">
        <p class="name">ANDREA BRIGLIADORI</p>
        <img src="PicturesOfMe/MeAtMIT.jpg" class="picture-of-me">
        <div class="further-info">
            <a href="https://www.linkedin.com/in/andrea-brigliadori-3abb8a248/" target="_blank">LinkedIn</a>
            <div>andrea.brigliadori.24@ucl.ac.uk</div>
            <a href="CV_AndreaBrigliadori.pdf" target="_blank">CV</a>
        </div>
    </div>
    <div class="elevator-pitch-and-papers-showcase">
        <div class="elevator-pitch">
            <p>Hello, my name is Andrea Brigliadori and I have recently submitted my thesis for the MSc in <i>AI for Biomedicine and Healthcare</i> at University College London. In this page, I present lines of research I would like to investigate, and some articles I have written. </p>
                <div class="elevator-pitch-row">
                    <div class="elevator-pitch-info">
                        <button class="dropdown-button" onclick=dropDown(13)>Research Interests</button>
                        <div id="13" class="dropdown-text">
                            I am interested in pursuing research in computational neuroimaging. I would like to employ machine learning to advance the state-of-the-art
                            with a particular focus on clinical applicability. My objective is to concretely contribute to the collective wellbeing through the knowledge acquired during my studies.
                        </div>
                    </div>
                </div>
                <div class="elevator-pitch-row">
                    <div class="elevator-pitch-info">
                        <button class="dropdown-button" onclick=dropDown(12)>Examples: two interesting projects</button>
                        <div id="12" class="dropdown-text">
                            1. Researchers at the CIBM MRI EPFL-AIT Section intend to map neurometabolism using magnetic resonance spectroscopic imaging. This approach captures information at the molecular scale and may advance our understanding, diagnosis, and treatment of neurological disorders.<br>
                            2. The DiCIPHR lab (University of Pennsylvania, Center for Biomedical Image Computing & Analytics) is proposing innovative solutions for quality assurance, harmonization, and analysis of diffusion MRI data, advancing clinical translation. 
                        </div>
                    </div>
                </div>
                <div class="elevator-pitch-row">
                    <div class="elevator-pitch-info">
                        <button class="dropdown-button" onclick=dropDown(11)>Topic of MSc thesis</button>
                        <div id="11" class="dropdown-text">
                            Improving the clinical applicability of a geometric deep learning model used for parameter estimation in diffusion MRI by enhancing its ability to generalise to unseen scanner settings.
                        </div>
                    </div>   
                </div>
                <div class="elevator-pitch-row">
                    <div class="elevator-pitch-info">
                        <button class="dropdown-button" onclick=dropDown(10)>Topic of BSc thesis</button>
                        <div id="10" class="dropdown-text">
                            Improving the biological reliability of statistical physics models for place cells.
                        </div>
                    </div>
                </div>
            
        </div>
        <div class="paragraph-title">Selected Articles</div>
        <div class="papers-showcase">
            <div class="papers-showcase-row">
                <p class="papers-showcase-column-name">Title</p>
                <p class="papers-showcase-column-name">Date</p>
            </div>
            <div class="papers-showcase-row">
                <div class="title-and-abstract">
                    <div class="paper-title">Geometric Deep Learning for Generalisable
Estimation of Microscopic Brain Tissue Properties
Across MRI Acquisition Settings (MSc thesis)</div>
                    <button class="dropdown-button" onclick=dropDown(6)>Abstract</button>
                    <div id="6" class="dropdown-text">
                        <p>This work focused on improving the ability of machine learning models to generalise across MRI scanner settings when estimating properties of brain microstructure. Microstructure refers to the tissue structure at the micrometre scale, and its properties are indicative of the health condition of the brain. The quantitative description of these properties can significantly accelerate the diagnosis of neurological disorders such as stroke and multiple sclerosis.
Capturing information on brain microstructure non-invasively from living human beings has been made possible by an advanced magnetic resonance imaging technique known as diffusion-weighted MRI. Using two main scanner settings, known as b-vectors and b-values, this technique provides images that can be leveraged by computational models to estimate microstructural properties. 
There is growing interest in obtaining these estimates using machine learning models, since they offer great efficiency advantages compared to traditional methods.
However, these models typically do not generalise well to changes of the scanner settings, and this limitation is contributing to hamper their clinical utility. Recently, a geometric deep learning model known as spherical convolutional neural network (SCNN) has been shown to maintain its performance despite variations of the b-vectors. Nonetheless, the model provides no performance guarantees for b-values unseen during training. The aim of this project is to enhance the generalisability of SCNNs to new b-values. A novel SCNN architecture that accepts b-values as input is proposed and shown to improve performance on both seen and unseen b-values, although further hyperparameter tuning may be needed to obtain stronger evidence.
                    </div>
                </div>
                <div class="paper-date">2025</div>
            </div>
            <div class="papers-showcase-row">
                <div class="title-and-abstract">
                    <div class="paper-title">Quantification of Hepatic Iron Overload using MRI</div>
                    <button class="dropdown-button" onclick=dropDown(5)>Abstract</button>
                    <div id="5" class="dropdown-text">
                        <p>This work focuses on quantifying iron overload in the liver from magnetic resonance images. Measuring
                        the amount of iron in the body is essential for the treatment of patients suffering from thalassemias, a
                        class of widespread genetic diseases compromising the production of red blood cells. Without frequent blood
                        transfusions, many of these patients do not reach the age of five. In turn, transfusions may lead to toxic
                        accumulations of iron, potentially inducing severe health complications such as cardiac failure. In order to
                        prevent this, it is critical to remove the excess iron in the body. At present, the only effective method is to
                        use medications able to bind iron and assist in its excretion. However, it is necessary to tailor the length and
                        intensity of this therapy to the specific patient, hence clinicians need to know the amount of surplus iron
                        to remove. To approach this problem, computational models have been developed and applied on magnetic
                        resonance images to estimate liver iron overload, which has been shown to closely relate to the total body
                        iron content. In this study, we contribute to the field with an in-depth analysis of one of these methods, obtaining promising estimation results.</p>
                        <a href="Quantification_of_Hepatic_Iron_Overload_using_MRI.pdf" target="_blank">Full text</a>
                    </div>
                </div>
                <div class="paper-date">2025</div>
            </div>
            <div class="papers-showcase-row">
                <div class="title-and-abstract">
                    <div class="paper-title">Machine Learning for Localising Proteins</div>
                    <button class="dropdown-button" onclick=dropDown(4)>Abstract</button>
                    <div id="4" class="dropdown-text">
                        Function and subcellular environment of a protein are tightly related. Because of this, localising a protein
                        can be of great utility in investigating its purpose. This, in turn, is a critical step in obtaining insights that
                        can guide tailored drug design for a disease. Currently, one of the gold standards for conducting protein
                        localisation is fluorescent microscopy imaging in a wet lab. However, the rapidly increasing amount of data
                        generated by proteomics demands more efficient strategies. In this context, machine learning poses itself as
                        an invaluable option to effectively complement the research conducted in the wet lab. We here pursue a
                        study on the application of machine learning methods to protein subcellular localisation, aiming to maintain
                        rigour and correctness throughout the work. The positive results obtained by our Voting Classifier confirm
                        potential of this interdisciplinary approach.
                    </div>
                </div>
                <div class="paper-date">2025</div>
            </div>
            <div class="papers-showcase-row">
                <div class="title-and-abstract">
                    <div class="paper-title">Building upon the Hopfield Model: the Statistical Physics of Place Cells (BSc thesis)</div>
                    <button class="dropdown-button" onclick=dropDown(3)>Abstract</button>
                    <div id="3" class="dropdown-text">
                        The modelling of place cells within the framework of statistical physics is here studied. Neurons are represented as binary units that can be organized in different orders, each corresponding to a specific cognitive map. Clusters of closely situated active neurons form "activity bumps" within the map that is then indicated as the one recalled by the network. During transition phases, these bumps may shift from one map to another. The nature of these transitions allows for the classification into distinct phases: one-bump, two-bump, spin glass, and paramagnetic.

                        The instance of an earlier attractor neural network is further developed by incorporating bioinspired components, and is used for a comparative analysis of the results. Specifically, a Poisson process, theta waves and an evolving external stimulus are integrated into the model, yielding novel interesting outcomes.
                    </div>
                </div>
                <div class="paper-date">2024</div>
            </div>
            <div class="papers-showcase-row">
                <div class="title-and-abstract">
                    <div class="paper-title">Protein-Aptamer Ranking Project</div>
                    <button class="dropdown-button" onclick=dropDown(2)>Abstract</button>
                    <div id="2" class="dropdown-text">
                        Biosensors are becoming an increasingly popular method of detecting specific chemical substances via the combination of a biological component and modern technologies.
                        Protein react in a unique way to aptamers, so certain aptamers can be used on a biosensor to capture specific proteins. The crucial point is the binding affinity between a protein-aptamer pair, which can be measured with Kd values, where lower figures indicate better affinity. In a traditional lab setting, testing
                        the binding affinity between one protein and aptamer pair proves too costly in terms of both time and money. This is where machine learning comes in. Given protein-aptamer complexes as input, machine learning models are able to rank them in order of binding
                        affinity. This implicitly helps identify which aptamers bind best to which proteins and may find applications in the search for a simpler and time-saving way to build biosensors able to detect proteins.
                    </div>
                </div>
                <div class="paper-date">2024</div>
            </div>
            <div class="papers-showcase-row">
                <div class="title-and-abstract">
                    <div class="paper-title">Image Reconstruction from fMRI Data</div>
                    <button class="dropdown-button" onclick=dropDown(1)>Abstract</button>
                    <div id="1" class="dropdown-text">
                        This research aims to enhance the understanding
                        and visualization of brain activity in conditions
                        like Amyotrophic Lateral Sclerosis (ALS) and severe neurological damage, which impair cognitive and motor functions. By developing methods to decode fMRI signals, we plan to visualize thoughts
                        and intended communications, aiding clinicians in ”reading” the thoughts of those unable to express themselves. This work also optimizes the ”Brain Diffuser” framework for use on standard computers, broadening accessibility and fostering innovation in neuroimaging.
                    </div>
                </div>
                <div class="paper-date">2024</div>
            </div>
            <div class="papers-showcase-row">
                <div class="title-and-abstract">
                    <div class="paper-title">AI Lab Project on Breast Cancer Cells</div>
                    <button class="dropdown-button" onclick=dropDown(0)>Abstract</button>
                    <div id="0" class="dropdown-text">
                        If it is true that there are still several major milestones AI needs to overcome to reach human-level, on the other hand it has nowadays made its way into the biological field, demonstrating its worth through innovative procedures and playing a critical role in the context of life sciences and health. Not by chance, the cost of a human genome sequence dropped from an estimated 1 million dollars in 2007, to 1000 dollars in 2014, and today it is approximately $600, while the time needed for vaccine development decreased from nearly a hundred years at the beginning of the 20th century, to the few months that provided us with the COVID vaccine.

                        In this research, we will employ large datasets about gene expression of cancerous cells maintained in two different conditions, hypoxia or normoxia. The overall aim is being able to understand and analyse the data through the implementation of machine learning algorithm and visualization methods, extracting from them the most important notions.

                        Finally, this information will be used to build a classifier, with the scope of answering a precise final question: can we identify hypoxic cells?
                    </div>
                </div>
                <div class="paper-date">2023</div>
            </div>
        </div>
    </div>

    <script src="Scripts/AB_PersonalWebsite.js"></script>
</body>
</head>
</html>
